Evaluator/TA/momentum_evaluator.py

```diff
@@ -74,6 +74,91 @@ def __init__(self):
                     self.set_eval_note((rsi_v[-1] - 100) / 200)
 
 
+# double RSI analysis
+class RSIWeightMomentumEvaluator(MomentumEvaluator):
+    DESCRIPTION = "Uses the Relative Strength Index (lengths of 14) to find dips and give them weight according to " \
+                  "the trend."
+
+    @staticmethod
+    def get_eval_type():
+        return Dict[str, int]
+
+    def _get_rsi_averages(self):
+        # compute the slow and fast RSI average
+        period_length = 14
+        slow_eval_count = 16
+        fast_eval_count = 4
```

Comment:
I think it can be better to create `RSIWeightMomentumEvaluator.json` and add this params as default in it.